lua/cmake/config.lua

```diff
@@ -19,7 +19,11 @@ local config = {
       only_on_error = false,
     },
     copy_compile_commands = true,
-    dap_configuration = { type = 'lldb', request = 'launch' },
+    dap_configurations = {
+      lldb_vscode = { type = 'lldb', request = 'launch' },
+      cppdbg_vscode = { type = 'cppdbg', request = 'launch', cwd = '${workspaceFolder}' },
```

Comment:
I just noticed that I forgot to remove `, cwd = '${workspaceFolder}'` before I commited.

I intended to remove it because I do not think it should be in the defaults since there already are defaults for `cwd`.

I had only added it temporarily because those defaults currently sets it to an empty string #51 which cppdbg_vscode does not accept.

Reply:
Fixed #51!